backend/tracim_backend/applications/share/lib.py

```diff
@@ -164,7 +165,7 @@ def get_content_share(self, content: Content, share_id: int) -> ContentShare:
                 .filter(ContentShare.content_id == content.content_id)
                 .filter(ContentShare.share_id == share_id)
                 .one()
-            )  # type: ContentShare
```

Comment:
this typing is not needed and create weird issue using black (AST related issue), that's the reason why i drop it.

Reply:
@inkhey this kind of comment must be in the code. No chance to read this MR when you are in the code

Reply:
This comment is just for reviewer to understand the diff, it make no sense to have this in code because it's clearly related to a bug in black, knowing that uneeded typing here create bug in a specific black version is not really helpful for dev.

Reply:
I disagree because a developer may see the code, then see there is no typing and would try to add it again. So something like: "do not add typing here - there is a bug in black https://www.github.com/black/back/xxx

Reply:
fixed by: 
- moving typing to a working place
- creating an issue in black : https://github.com/psf/black/issues/1329
- adding comment about black issue